### ISSUE TITLE
About WaveRNN, not Alternative.

### ISSUE THREAD

**Issue by bjtommychen** [fatchord/WaveRNN#23]
Hi,
    The Alternative one is easy to train, and got good result.
    I also trained the WaveRNN like your NB1/2/3 . but quality is not good even after 7 days.  loss >5.  I  compare it with your Alternative , I wonder if add NN upsampling or Resnet will make it better.  Would you share your thoughts. thanks.

**Comment by fatchord** [comment]
@bjtommychen I haven't conditioned the first three notebooks on spectrograms. Did you implement conditioning yourself?

**Comment by kubicndmr** [comment]
@bjtommychen can you show how you conditioned WaveRNN by yourself

**Comment by bjtommychen** [comment]
like this:
        input = torch.cat((prev_y, mels), dim=1)
        coarse_input_proj = self.I_coarse(input)
...
        fine_input = torch.cat([prev_y, mels, current_coarse], dim=1)
...

**Comment by hdmjdp** [comment]
@bjtommychen 3q

**Comment by bjtommychen** [comment]
@hdmjdp same as me. the origin model's quality is worse than Alternative one. and also training slower.

**Comment by ohleo** [comment]
@bjtommychen 
Could you share the synthesized sample of 16-bit WaveRNN and training sequence length(=seq_len in notebook3)?
I think training sequence length should be 7936 samples(0.359 sec in 22.05kHz data) like r9y9 wavenet code. But, Training-time will be unrealistic.
How many training step is needed for intelligible sound?

I would appreciate your reply.
